### PR TITLE
feat(i18n): localize groups

### PR DESCRIPTION
**Pull request by pavanjoshi914**
Localize Groups module
use form helpers wherever simple HTML form tags used

### Screenshots of the changes (If any) -
image
image
image
image
image
image
image
image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: config/locales/views/groups/hi.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    remove_assignment_message: "क्या आप वाकई इस असाइनमेंट को डिलीट करना चाहते हैं?"
+    remove_member_message: "क्या आप वाकई इस सदस्य को हटाना चाहते हैं?"
+    edit:
+      title: "%{name} - समूह का नाम एडिट करें"
+      main_heading: "समूह का नाम एडिट करें"
+    new:
+      title: "सर्किटवर्स - नया समूह"
+      main_heading: "नया समूह"
+      main_description_html: "छात्रों के लिए प्रोजेक्ट सेट करने और ग्रेड देने के लिए मेंटर्स द्वारा समूहों का उपयोग किया जा सकता है।
+         आप डॉक्यूमेंटेशन <a class='anchor-text' href='https://docs.circuitverse.org/#/groups' target='_blank'>यहां</a> पा सकते हैं।"
```
Comment (Shivansh2407) [inline config/locales/views/groups/hi.yml:15]: पा -> पढ़

**Review by Shivansh2407 — APPROVED** [review]
Looks good to me @pavanjoshi914 . Please have a look at the minor change, I've suggested. Thanks! :tada:

File: config/locales/views/groups/hi.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        assignment_table_head:
+          name: "नाम"
+          deadline: "समय सीमा"
+          grading: "ग्रेडिंग"
+          actions: "एक्शन्स"
+          grade: "ग्रेड"
+        time_remaining_html: "<strong>शेष समय:</strong> %{time_remaining}"
+        start_working_btn: "काम शुरू करे"
+        see_work_done: "अपना काम देखें"
+        not_submitted: "सबमिट नहीं किया गया"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/groups/hi.yml:33]: Replace सबमिट with प्रस्तुत

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
Good work @pavanjoshi914 🎉 I have suggested few changes, please have a look at them.

File: config/locales/views/groups/hi.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+hi:
+  groups:
+    add_member_description: "कॉमा/स्पेस से अलग या अलग-अलग लाइन्स में ईमेल आईडी दर्ज करें। यदि उपयोगकर्ता रजिस्टर्ड नहीं हैं, तो उन्हें साइन अप करने का अनुरोध करते हुए एक ईमेल आईडी भेजी जाएगी।"
```
Comment (ShreyaPrasad1209) [inline config/locales/views/groups/hi.yml:3]: Add the same changes here as I have suggested in #2425

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]

**Comment by pavanjoshi914** [comment]
did required changes @ShreyaPrasad1209

**Review by ShreyaPrasad1209 — APPROVED** [review]

**Pull request merged**